nominations/migrations/0056_auto_20171002_2054.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Same migration prefix 0056_* as another PR https://github.com/Our-Revolution/site/pull/20/files#diff-6e73b9542001040d0afd65bc23e2bf14R1

Is this something we need to address or avoid? Or do we not care if prefix is the same as long as the full timestamp is different?